packages/kotti-ui/source/kotti-navbar/KtNavbar.vue

```diff
@@ -55,7 +55,7 @@
 import { computed, defineComponent, provide, ref } from '@vue/composition-api'
 import { mixin as clickaway } from 'vue-clickaway'
 
-import { Kotti } from '../types'
+import { propValidator } from '../props'
 
 import NavbarLogo from './components/NavbarLogo.vue'
 import NavbarMenu from './components/NavbarMenu.vue'
```

Comment:
can't comment on the line below

create an issue to delete `NavbarNotification` and the prop `notifications` on the `KtNavbar` 
as part of the next published release of Kotti

Reply:
@carsoli TODO tomorrow